assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -54,8 +63,12 @@ function useUploadVideoFrame() {
         featured_media: posterId,
         post: storyId,
       });
-      const newState = { posterId, poster };
-      setProperties(elementId, newState);
+      if (elementId) {
```

Comment:
This no longer really works. We need to remove `elementId` from this method entirely and instead call an action on the story reducer to update all elements that depend on this resource. Alternatively we'd need to track an array of dependent elements, but I think story reducer would be simpler.

Reply:
I like this idea, but I think this is out of scope of this PR. I will create a follow up ticket.

Reply:
New issue #615